swiftwinrt/Resources/Support/IUnknownRef.swift

```diff
@@ -1,45 +1,21 @@
 // Copyright © 2021 Saleem Abdulrasool <compnerd@compnerd.org>
 // SPDX-License-Identifier: BSD-3
 
-import WinSDK
-
 @_fixed_layout
 public final class IUnknownRef {
```

Comment:
Is there a difference between `IUnknownRef` and `ComPtr<C_IUnknown>` at this point? Do we need to keep both?

Reply:
no, we could likely get rid of `IUnknownRef`

Reply:
but i'll do that in a separate PR since that requires some code changes in Arc